services/ui_backend_service/api/tag.py

```diff
@@ -0,0 +1,13 @@
+from services.utils import handle_exceptions, web_response
+
+
+class TagApi(object):
+    def __init__(self, app, db):
+        self.db = db
+        app.router.add_route("GET", "/tags", self.get_all_tags)
```

Comment:
is this route used by the UI in its current form?

Reply:
It is not